test/unit/provisioner/test_ansible_playbook.py

```diff
@@ -27,7 +27,9 @@
 
 @pytest.fixture
 def _instance(config_instance):
-    return ansible_playbook.AnsiblePlaybook('playbook', config_instance)
+    _instance = ansible_playbook.AnsiblePlaybook('playbook', config_instance)
+    _instance._config.state.change_state('sanity_checked', True)
```

Comment:
My only concern here is that you did not include a comment explaining why this is needed. As the reason is quite well hidden, reading these lines makes anyone wonder why their were added. Using git blame to dig is not always convenient.

Reply:
Fair point. And I am somehow reconsidering my placement of where I trigger the sanity checks (causing this issue ...) but have not had really time to take a proper look.

Reply:
OK, something more informative in https://github.com/ansible/molecule/pull/2068/commits/4fed8fc2f3e4b09c4b1f6cee8652d2c0817aab0b.